schemas/triniti/apollo/mixin/reactions/latest.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:triniti:apollo:mixin:reactions:1-0-0" mixin="true" extends="gdbots:ncr:mixin:node:v1">
+    <fields>
+      <!--
```

Comment:
@bhsiao55 I'd just do a reactions int map.  since we're not forcing the reaction types at the schema level.

Reply:
@bhsiao55 add a target string type format slug.  so we know what reactions are referring to... i.e. article, video, gallery, etc.  the node ref of the reactions is not enough to derive the target.